### DISCUSSION TITLE
Maven <latest> and version scanning

### DISCUSSION THREAD

**Discussion by bmaginni** [renovatebot/renovate#21956] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

35.61.0

### If you're self-hosting Renovate, select which platform you are using.

Bitbucket Server

### If you're self-hosting Renovate, tell us what version of the platform you run.

n/a

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Is there any way to limit what versions renovate tries to "find"
I think it scans the maven-metadata.xml file and then iterates thru all versions
Can we reduce that scan to limit what it attempts to check.
Can it for example just use <latest> or latest-5 versions etc?

<metadata modelVersion="1.1.0">
<groupId>org.springframework</groupId>
<artifactId>spring-webmvc</artifactId>
<versioning>
<latest>6.0.8</latest>
<release>6.0.8</release>
<versions>
<version>1.0-rc1</version>
<version>1.0</version>
<version>1.0.1</version>
<version>1.1-rc1</version>
<version>1.1-rc2</version>
<version>1.1</version>
<version>1.1.1</version>
<version>1.1.2</version>
<version>1.1.3</version>
<version>1.1.4</version>
<version>1.1.5</version>
.......
<version>5.3.26</version>
<version>5.3.27</version>
<version>6.0.4</version>
<version>6.0.5</version>
<version>6.0.6</version>
<version>6.0.7</version>
<version>6.0.8</version>
</versions>

### What you tried so far.

I've tried looking thru documentation and discussions

### Relevant debug logs

<details><summary>Logs</summary>

```
DEBUG: HEAD https://mavenmaster.*******.****.com/repository/****_Snapshot_Group/org/springframework/spring-webmvc/2.0.6/spring-webmvc-2.0.6.pom = (code=ERR_NON_2XX_3XX_RESPONSE, statusCode=404 retryCount=0, duration=3224) (repository=CM/repo, baseBranch=develop)
DEBUG: HEAD https://mavenmaster.*******.****.com/repository/****_Snapshot_Group/org/springframework/spring-webmvc/2.0.5/spring-webmvc-2.0.5.pom = (code=ERR_NON_2XX_3XX_RESPONSE, statusCode=404 retryCount=0, duration=3254) (repository=CM/repo, baseBranch=develop)
```

</details>

**Comment by rarkins** [comment]
Check `RENOVATE_EXPERIMENTAL_NO_MAVEN_POM_CHECK` in the docs if you're happy to skip all pom checking

**Comment by bmaginni** [comment]
Thanks, this worked well.
Are there any other configuration options to maybe scan the <latest> - 10 versions as opposed to all listed in the maven-metadata.xml?

**Reply by rarkins** [reply]
No, there's not
